.github/workflows/build_all.yml

```diff
@@ -272,6 +272,103 @@ jobs:
           SLACK_ICON_EMOJI: ":heavy_check_mark:"
           SLACK_COLOR: "#00FF00"
           SLACK_FOOTER: ' '
+  agw-container-build:
```

Comment:
Is there a way to add multiple labels to the deployed images? If not labels, any other form of annotation? My hope is to be able to see images built over a number of days (ex: last 7 days) that we can then use for nightly builds and tests.

Reply:
we could add labels to the docker images. What labels exactly would you want?

Reply:
Can we add commit ids? My hope is to identify which build this is based on.

Reply:
It's already tagging with short sha and latest